### PR TITLE
[Ready] Remove zelnode & spork / Update zelnode tier structure

### PR DESCRIPTION
**Pull request by blondfrogs**
27cb6ce087517ef58eaa26aa87014c196d8513fa - Removed old non deterministic zelnode and spork code 

30bf7f9769d3b5b02f0125c206d8c5b366300ada
3e35a50390b4d71e85fdb6c2ba2f730e675649a4
8ae9d4193c7bacbb5735c025e233017ef7abf18f
78651bb5f5091a67f77752abc589fd4a5a78117b - In the current code, doing work on the tiers was completed by selecting them one at a time and performing the same task. This caused the following problems, duplicate code, and make it hard to add or remove tiers. 

- With these tier structure changes, there will only be a couple places in the codebase that will require changes if and when we decide to add/remove a tier structure. 

**TLDR**
- Removed unused zelnode and spork code
- Removed duplicate tier code

### CODE REVIEW

**Comment by Cabecinha84** [comment]
Can we remove this from debug logs:
- AcceptToMemoryPool: Adding zelnode transaction to mempool:
- remove: Remove zelnode tx from mempool
We have gb of logs because of this lines, are they still needed?

**Review by TheTrunk — APPROVED** [review]

**Comment by blondfrogs** [comment]
> Can we remove this from debug logs:
> 
>     * AcceptToMemoryPool: Adding zelnode transaction to mempool:
> 
>     * remove: Remove zelnode tx from mempool
>       We have gb of logs because of this lines, are they still needed?

Moved logs to specific -debug=zelnode now

**Review by Cabecinha84 — APPROVED** [review]
Ack
